Arrowgene.Ddon.GameServer/Handler/CraftStartEquipGradeUpHandler.cs

```diff
@@ -75,48 +75,86 @@ public override S2CCraftStartEquipGradeUpRes Handle(GameClient client, C2SCraftS
                     throw new ResponseErrorException(ErrorCode.ERROR_CODE_ITEM_INVALID_ITEM_NUM, "Client Item Desync has Occurred.");
                 }
             }
-
+            uint addEquipPoint = (uint)((isGreatSuccess ? 300 : 180) * (0.8 + (Random.Shared.NextDouble() * 0.4)));
             // TODO: make use of pawn crafting skills instead
-            // Update item equip points in the database
-            uint addEquipPoint = (uint)((IsGreatSuccess ? 300 : 180) * (0.8 + (Random.Shared.NextDouble() * 0.4)));
             currentTotalEquipPoint += addEquipPoint;
 
             // Subtract less Gold if support pawn is used and add slightly more points
             if (request.CraftSupportPawnIDList.Count > 0)
             {
                 goldRequired = (uint)(goldRequired * 0.95);
-                currentTotalEquipPoint += 10;
+                currentTotalEquipPoint = (uint)(currentTotalEquipPoint * 1.5); // Fake stuff until pawn craft levels
```

Comment:
Better put TODO: before the comment so its easier to search for TODOs in the proejct